webpack.dev.js

```diff
@@ -2,5 +2,4 @@ const common = require('./webpack.common');
 const merge = require('webpack-merge');
 
 module.exports = merge(common, {
```

Comment:
Merging with an empty object?

Reply:
Yes, configuration for `dev` now equals the one from `common`, but with this empty object you have everything set up to change that state.

E.g. in the near future introduction of tests may need different config. Etc.